src/System.Windows.Forms/src/System/Windows/Forms/DataGridView.cs

```diff
@@ -2357,17 +2357,21 @@ public DataGridViewCellStyle DefaultCellStyle
                     CellStyleChangedEventArgs.ChangeAffectsPreferredSize = (dgvcsc == DataGridViewCellStyleDifferences.AffectPreferredSize);
                     OnDefaultCellStyleChanged(CellStyleChangedEventArgs);
                 }
+
+                DefaultCellStyleHaveSetByUser = true;
             }
         }
 
+        internal bool DefaultCellStyleHaveSetByUser { get; set; } = false;
```

Comment:
I'm struggling to understand the meaning of this property. 
Is it to indicate that a user defined own cell style? If so, should we call it as follows?
```suggestion
        /// <summary>
        ///  Indicates whether ...... <-- ???
        /// </summary>
        internal bool UseCustomCellStyle { get; set; } = false;
```

Reply:
Yes. I fixed this property, thanks.